enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}{% load enterprise_string_utils %}<html>
+<body>
+<p>{% if user_name %}{% blocktrans %}Dear {{ user_name }},{% endblocktrans %}{% else %}{% blocktrans %}Hi!{% endblocktrans %}{% endif %}</p>
+<p>{% if enrolled_in.type == "program" %}
+{% blocktrans with program_url=enrolled_in.url program_name=enrolled_in.name program_branding=enrolled_in.branding|pad_right start_date=enrolled_in.start|date:"F j, Y" %}You have been enrolled in <a href="{{ program_url }}">{{ program_name }}</a>, a {{ program_branding }}program offered by {{ organization_name }}. This program begins {{ start_date }}. For more information, see <a href="{{ program_url }}">{{ program_name }}</a>.{% endblocktrans %}{% else %}
```

Comment:
For the HTML template, you don't need `pad_right` - just put a space in `{{ program_branding }}program`. Browsers automatically combine adjacent whitespace, so the end result is the same. I'd probably do the same for the plaintext too - it's simpler and worst case scenario is that some program with no branding name shows up as "a  program" and mode people won't even notice the extra space. Many modern mail clients render plain text emails using an HTML interface anyways, which means that adjacent spaces get combined there too.